src/js/modules/calculator.js

```diff
@@ -0,0 +1,41 @@
+import { CALC_INPUT_TYPE } from '../constant'
+
+export default class Calculator {
+  constructor(props) {
+    this.limit = props.limit
+    this.x = ''
+    this.y = ''
+    this.operator = ''
+  }
+
+  validateInput(type) {
+    if (!this.x) return type !== CALC_INPUT_TYPE.DIGIT ? false : true
+    if (this.x.length === this.limit && !this.operator) return type !== CALC_INPUT_TYPE.OPERATION ? false : true
+    if (this.operator && !this.y) return type !== CALC_INPUT_TYPE.DIGIT ? false : true
+    if (this.y.length === this.limit) return false
+    return true
+  }
+
+  setInput({ type, value }) {
+    if (type === CALC_INPUT_TYPE.DIGIT) !this.operator ? (this.x += value) : (this.y += value)
+    else this.operator = value
+    return this
```

Comment:
이렇게 줄여쓰는건 지양하면 좋아요!
웬만하면 풀어써야 코드를 읽기가 더 수월하답니다 ㅎㅎ
```suggestion
  setInput({ type, value }) {
    if (type === CALC_INPUT_TYPE.DIGIT) {
      const selectedInput = !this.operator ? 'x' : 'y';
      this[selectedInput] += value;
      return this;
    }
    this.operator = value;
    return this
```
이렇게 표현해주면 어떨까요?